tests/EalTest.cpp

```diff
@@ -142,3 +142,37 @@ TEST(SplineInterpTest, Extrapolate) {
   EXPECT_THROW(eal::splineInterpolate(data, times, 4.0),
                invalid_argument);
 }
+
+TEST(PoisitionCoeffTest, ValidInput) {
+  double time = 2.0; 
+  vector<vector<double>> coeffs = {{3.0, 2.0, 1.0},
+                                   {2.0, 3.0, 1.0},
+                                   {1.0, 2.0, 3.0}};
```

Comment:
Would we maybe want to check negative coefficient values...?

Reply:
@SgStapleton :+1: